src/components/prev-next-buttons/prev-next-buttons.js

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import {
+  Button,
+  Icon,
+} from '@folio/stripes/components';
+
+import { PAGE_SIZE } from '../../constants';
+
+import styles from './prev-next-buttons.css';
+
+const PrevNextButtons = ({
+  page,
+  setPage,
+  totalResults,
+  isLoading,
+}) => {
+  const intl = useIntl();
+
+  const maxItemsLength = page * PAGE_SIZE;
+  const disabledNext = maxItemsLength > totalResults;
```

Comment:
Should this be `>=`? If we have equal total results and currently displayed results I think we should disable the button also